.eslintrc.json

```diff
@@ -65,7 +70,14 @@
     // Because we want imports sorted!
     // The built-in "sort-imports" doesn't seem to be sufficient, and the
     // "eslint-plugin-import" doesn't seem to work.
-    "simple-import-sort/sort": "warn"
+    "simple-import-sort/sort": "warn",
+
+    // Disable warning:
+    // "React Hook useEffect has a missing dependency: ... Either
+    // include it or remove the dependency array"
+    // However, this kind of warning seems to be useful when working with hooks (performance optimizations)
+    // https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies
+    "react-hooks/exhaustive-deps": 0
   },
```

Comment:
I don't think this is a good idea ... we are violating it all over the place, which is annoying, but it's usually for a reason that involves some tech debt we have incurred by moving so fast. Having the warnings (or explicit overrides to suppress them) is a good thing IMO

Reply:
FWIW the usual suspect for this warning in our codebase is a function that has to be defined within the render function; it turns out that the `useCallback` hook can help with that (https://reactjs.org/docs/hooks-reference.html#usecallback)

There are a few examples of us using it to help with this issue now, you can see it e.g. here: https://github.com/Recidiviz/covid19-dashboard/blob/master/src/hooks/useError.tsx#L6

Reply:
reverted, and implemented `useCallback`. Thanks!